File: sdks/thunkmetrc-go/wrapper/models/delivery_package_requiredlabtestbatch.go
package models

type DeliveryPackageRequiredlabtestbatch struct {
    LabTestBatchId int64 `json:"LabTestBatchId,omitempty"`
    LabTestBatchName string `json:"LabTestBatchName,omitempty"`
    PackageId int64 `json:"PackageId,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/DeliveryPackageRequiredlabtestbatch.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class DeliveryPackageRequiredlabtestbatch(
    @JsonProperty("LabTestBatchId")
    val labTestBatchId: Int? = null,
    @JsonProperty("LabTestBatchName")
    val labTestBatchName: String? = null,
    @JsonProperty("PackageId")
    val packageId: Int? = null
)
